db/migrate/20230125103106_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.integer :PostCounter
```

Comment:
- OPTIONAL ] I highly recommend to change the names here to be posts_counter since this is best practice when naming the column in Ruby on rails so I think you can add new migration to change the name of the column🙏🏻💯✨✅✅🔥